ament_copyright/ament_copyright/parser.py

```diff
@@ -178,7 +178,8 @@ def search_copyright_information(content):
     year_range = '%s-%s' % (year, year)
     year_or_year_range = '(?:%s|%s)' % (year, year_range)
     pattern = r'^[^\n\r]?\s*(?:\\copyright\s*)?' \
-              r'Copyright(?:\s+\(c\))?\s+(%s(?:,\s*%s)*),?\s+([^\n\r]+)$' % \
+              r'(?:Copyright|COPYRIGHT|copyright)' \
```

Comment:
Maybe you just want to make it case insensitive? Or perhaps it's better for it to be one of these cases.

I don't feel strongly about this. Here's a reference: https://stackoverflow.com/a/500870